k8s/service_controller/server/service_storage.go

```diff
@@ -12,21 +12,38 @@ type ServiceBroker struct {
 	Password string `json:password""`
 }
 
-type ServiceStorage interface {
+// The Broker interface provides functions to deal with brokers.
+type Broker interface {
 	ListBrokers() ([]*ServiceBroker, error)
 	GetBroker(string) (*ServiceBroker, error)
 	GetInventory(string) (*model.Catalog, error)
 	AddBroker(*ServiceBroker, *model.Catalog) error
 	DeleteBroker(string) error
+}
 
+// The Servicer interface provides functions to deal with services.
+type Servicer interface {
```

Comment:
I'd prefer if `Servicer` was based on the word `Instance` instead of `Service` because the diff between a service and service broker is thin to a newbie and to avoid confusion I'd prefer to use the terms we use in other spots: ServiceBroker, ServiceInstance and ServiceBinding.